src/Common/src/TypeSystem/Serialization/FieldDesc.Serialization.cs

```diff
@@ -0,0 +1,33 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Internal.TypeSystem
+{
+    // Additional members of FieldDesc related to serialization.
+    partial class FieldDesc
+    {
+        /// <summary>
+        /// Gets a value indicating whether this field is not serialized.
+        /// specially.
+        /// </summary>
+        public virtual bool IsNotSerialized
```

Comment:
It does not feel 100% right to have these serialization bits on the type system (the alternative would be extension method). I do not have a strong opinion about it. I am ok with it if you are ok with it.

Reply:
I look at them same as the interop one. Could be avoided, but this ends up being more convenient.